src/views/Receive.vue

```diff
@@ -98,6 +98,18 @@ export default {
     routeSource () {
       return this.$route.query.source || null
     },
+    prefix() {
+      return ({
```

Comment:
Can it reuse the `chainName` property? 
Just concious of adding more places that we need to update for integrations :D